.travis.yml

```diff
@@ -26,7 +26,9 @@ env:
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath bleach'
-        - PIP_DEPENDENCIES=''
+        # For testing purposes, before these packages are available on pypi
+        - GIT_DEPENDENCIES='git+https://github.com/astropy/pytest-doctestplus.git git+https://github.com/astropy/pytest-remotedata.git git+https://github.com/astropy/pytest-openfiles.git'
```

Comment:
Does this even work? I never done it like this before. To install a package straight from repo, I use an incantation like https://github.com/spacetelescope/stsynphot_refactor/blob/master/.travis.yml#L103

Reply:
Well, the idea was that if `GIT_DEPENDENCIES` gets expanded within `PIP_DEPENDENCIES` properly, then pip knows what to do with those `git+ ...` strings. But it's apparently not working so I'll have to try something else.